selenium_tests/cross_listing/remove_cross_listing_tests.py

```diff
@@ -0,0 +1,56 @@
+from ddt import ddt, data, unpack
+
+from selenium_common.base_test_case import get_xl_data
+
+from selenium_tests.cross_listing.cross_listing_base_test_case import \
+    CrossListingBaseTestCase, TEST_DATA_CROSS_LISTING_MAPPINGS
```

Comment:
I wonder if, in the future, this could just be a property of the base test case class, so it can be accessed via `cls.test_data_xls`... just thinking out loud. It's fine the way it is (and has been all along), just more verbose this way.